pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -409,19 +409,21 @@ func (m *manager) canAdmitPod(allocatedPods []*v1.Pod, pod *v1.Pod) (bool, strin
 func (m *manager) canResizePod(allocatedPods []*v1.Pod, pod *v1.Pod) (bool, string, string) {
 	// TODO: Move this logic into a PodAdmitHandler by introducing an operation field to
 	// lifecycle.PodAdmitAttributes, and combine canResizePod with canAdmitPod.
-	if v1qos.GetPodQOS(pod) == v1.PodQOSGuaranteed && !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScalingExclusiveCPUs) {
-		if m.containerManager.GetNodeConfig().CPUManagerPolicy == "static" {
+	if v1qos.GetPodQOS(pod) == v1.PodQOSGuaranteed {
+		if !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScalingExclusiveCPUs) &&
+			m.containerManager.GetNodeConfig().CPUManagerPolicy == "static" &&
```

Comment:
nit: is `static` for cpu and `Static` for memory intentionally cased differently (lower case vs camel cased)? I noticed that it's that way both here and in your tests. If so, wondering if they should be consts, mostly because I find it a bit odd / hard to remember that they are cased differently, and I see you've had to write out the strings twice in this PR (here and in the tests)